bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Extensions/ComputeChaumPedersenProofs.cs

```diff
@@ -0,0 +1,211 @@
+using ElectionGuard.Ballot;
+using ElectionGuard.Decryption.Accumulation;
+using ElectionGuard.Decryption.Challenge;
+using ElectionGuard.Decryption.ChallengeResponse;
+using ElectionGuard.Decryption.Tally;
+
+namespace ElectionGuard.Decryption.Extensions;
+
+/// <summary>
+/// Extensions for computing Chaum-Pedersen proofs
+/// </summary>
+public static class ComputeChaumPedersenProofs
```

Comment:
just like the other static extension-based classes, most of this is just an iterator and the interesting bits are in the selection